cmd/fmt.go

```diff
@@ -209,5 +219,6 @@ func init() {
 	formatCommand.Flags().BoolVarP(&fmtParams.overwrite, "write", "w", false, "overwrite the original source file")
 	formatCommand.Flags().BoolVarP(&fmtParams.list, "list", "l", false, "list all files who would change when formatted")
 	formatCommand.Flags().BoolVarP(&fmtParams.diff, "diff", "d", false, "only display a diff of the changes")
+	formatCommand.Flags().BoolVarP(&fmtParams.fail, "fail", "f", false, "non zero exit code on reformat")
```

Comment:
💭 Do we need the short parameter `-f`? It's just that one-letter parameter choices are rare and we'd had to break some compatibility with re-purposing `-t` on `opa test` earlier this year... 🤔

Reply:
I'm fine with removing it. Was just trying to stay in style